platform/src/test/java/platformstepdefinition/PF_Header.java

```diff
@@ -0,0 +1,24 @@
+package platformstepdefinition;
+
+import org.junit.runner.RunWith;
+import org.openqa.selenium.WebDriver;
+
+import base.NewBaseClass;
+import cucumber.api.java.en.Given;
+import cucumber.api.junit.Cucumber;
+import platform.pageobjects.Header.Header;
+
+@RunWith(Cucumber.class)
+public class PF_Header extends NewBaseClass {
+
+	public WebDriver driver;
```

Comment:
Replace with public WebDriver driver = ObjectHelper.driver; 
Needed to retain driver instance

Reply:
Done Changes as requested